grails-app/conf/templates/configuration/products.csv

```diff
@@ -1,58 +1,58 @@
-,VX576,,"Printer, Multifunction, Color, HP Inkjet PageWide Pro 477dw",IT Equipment,,,Each,,1229,,,,,,,,,,,,,,,,
-,XJ173,,"Batteries, Non Rechargeable, AA",Office Supplies,,,Each,,1.5,,,,TRUE,,,,,,,,,,,,
-,RU651,,"Batteries, Non Rechargeable, AAA",Office Supplies,,,Each,,1.5,,,,TRUE,,,,,,,,,,,,
-,XJ252,,"Binder clips, 19mm",Office Supplies,,,Each,,1,,,,,,,,,,,,,,,,
-,RS138,,"Binder clips, 25mm",Office Supplies,,,Each,,1,,,,,,,,,,,,,,,,
-,MV069,,"Binder clips, 32mm",Office Supplies,,,Each,,1,,,,,,,,,,,,,,,,
-,FF240,,"Envelope, A3, Manila",Office Supplies,,,Each,Mail Room,0.01,,,,,,,,,,,,,,,,
-,DD108,,"Envelope, A3, White",Office Supplies,,,Each,Mail Room,0.01,,,,,,,,,,,,,,,,
-,GT068,,Staples 23/13 pack of 1000,Office Supplies,,,Each,Mail Room,3,,,,,,,,,,,,,,,,
-,PH950,,Staples 23/17 pack of 1000,Office Supplies,,,Each,Mail Room,3,,,,,,,,,,,,,,,,
-,CG819,,"File cabinet, Lateral, 5-drawer",Furniture and Equipment,,,Each,,55,,,,,,,,,,,,,,,,
-,UQ102,,"File cabinet, Lateral, 6-drawer",Furniture and Equipment,,,Each,,55,,,,,,,,,,,,,,,,
-,MR962,,"Stool, rolling",Furniture and Equipment,,,Each,,145.96,,,,,,,,,,,,,,,,
-,SZ449,,"Desk, Office, Wooden or metal",Furniture and Equipment,,,Each,,250,,,,,,,,,,,,,,,,
-,MG983,,"Chair, Office, Rolling",Furniture and Equipment,,,Each,New Employee Package,25,,,,,,,,,,,,,,,,
-,RZ479,,"Chair, Plastic",Furniture and Equipment,,,Each,,100,,,,,,,,,,,,,,,,
-,HP196,,"Beaker, Graduated, Borosilicate glass, 100mL",Lab,,,Each,Tertiary Care,1.5,,,,,,,,,,,,,,,,
-,MY523,,"Beaker, Graduated, Borosilicate glass, 250mL",Lab,,,Each,Tertiary Care,2.35,,,,,,,,,,,,,,,,
-,BX996,,"Beaker, Graduated, Borosilicate glass, 400mL",Lab,,,Each,Tertiary Care,4.05,,,,,,,,,,,,,,,,
-,BF640,,"Coat, Laboratory, Large, Unisex",Lab,,,Each,Tertiary Care,20,,,,,,,,,,,,,,,,
-,QS823,,"Coat, Laboratory, Medium, Unisex",Lab,,,Each,Tertiary Care,20,,,,,,,,,,,,,,,,
-,SJ489,,"Coat, Laboratory, Small, Unisex",Lab,,,Each,Tertiary Care,20,,,,,,,,,,,,,,,,
-,PK000,,"Ethanol (ethyl alcohol), 95%, reagent grade, 16oz",Lab,,,Each,Tertiary Care,4.71,,,,TRUE,,,,,,,,,,,,
-,HG966,,"Tube, Blood collection, K2 EDTA, Lavender top, Plastic, 4mL",Lab,,,Each,Tertiary Care,0.068,TRUE,,,,,,,,,,,,,,,
-,DJ064,,"Tube, Blood collection, K2 EDTA, Whole blood, Lavender, 2mL (pediatric)",Lab,,,Each,Tertiary Care,0.068,TRUE,,,,,,,,,,,,,,,
-,TN940,,"Tube, Blood collection, Serum, Red, Plastic, 3mL (pediatric)",Lab,,,Each,Tertiary Care,0.0692,TRUE,,,,,,,,,,,,,,,
-,KY133,,"Tube, Blood collection, Serum, Red, Plastic, 4mL",Lab,,,Each,Tertiary Care,0.0845,TRUE,,,,,,,,,,,,,,,
+Id,ProductCode,ProductType,Name,Category,GLAccount,Description,UnitOfMeasure,Tags,UnitCost,LotAndExpiryControl,ColdChain,ControlledSubstance,HazardousMaterial,Reconditioned,Manufacturer,BrandName,ManufacturerCode,ManufacturerName,Vendor,VendorCode,VendorName,UPC,NDC,Created,Updated
+,QX039,Default,Morphine 10mg immediate release tablet,Pain,,,Each,Tertiary Care,0.25,TRUE,,TRUE,,,,,,,,,,,,,
```

Comment:
Why is this lower case? I would have assumed we'd either use the id or code field, both values in those fields should be capitalized. If we're using the ProductType.name, that would be less ideal.

Reply:
The name is used here. It is unique too.